dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/startup/InitTableAlteror.java

```diff
@@ -138,6 +138,10 @@ public void execute()
             "'F_TRACKED_ENTITY_DATAVALUE_DELETE', 'F_TRACKED_ENTITY_DATAVALUE_READ', 'F_VIEW_EVENT_ANALYTICS', " +
             "'F_TRACKED_ENTITY_INSTANCE_SEARCH', 'F_TRACKED_ENTITY_INSTANCE_ADD', 'F_TRACKED_ENTITY_INSTANCE_DELETE'," +
             "'F_PROGRAM_ENROLLMENT', 'F_PROGRAM_UNENROLLMENT', 'F_PROGRAM_ENROLLMENT_READ', 'F_IMPORT_GML', 'F_SQLVIEW_MANAGEMENT');" );
+
+        executeSql( "UPDATE dataelementcategoryoption SET publicaccess = 'rwrw----' WHERE uid = 'xYerKDKCefk'" );
+        executeSql( "UPDATE dataelementcategory SET publicaccess = 'rwrw----' WHERE uid = 'GLevLNI9wkl'" );
+        executeSql( "UPDATE categorycombo SET publicaccess = 'rwrw----' WHERE uid = 'bjDvmb4bfuf'" );
     }
```

Comment:
same as above, only categoryOption needed this. 
uid can be different between databases, I think we need to use the name "default" here.